package.json

```
@@ -9,6 +9,8 @@
     "lint:fix": "next lint --max-warnings 0 --fix",
     "format": "prettier --write --loglevel=warn '*/**/*.{js,jsx,ts,tsx,html,gql,graphql,json}'",
     "fix": "yarn format && yarn lint:fix",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook --output-dir ./build",
```

たぶんこうしないとstorybookの画像が表示できないと思います！

```suggestion
    "storybook": "start-storybook -p 6006 -s ./public",
    "build-storybook": "build-storybook -s public",
```

---

確かにです。
修正しました！ f552b97